wcomponents-theme/src/main/xslt/wc.ui.fieldindicator.xsl

```diff
@@ -6,13 +6,22 @@
 		Transform for ui:fieldindicator which is output of WFieldWarningIndicator.
 	-->
 	<xsl:template match="ui:fieldindicator">
```

Comment:
For consistency, should this have been fieldIndicator.xsl? But looking at the other xsl there are different camel cases.

Reply:
See previous comment about git being case insensitive for file names!